AK/Math.h

```diff
@@ -220,6 +220,25 @@ constexpr T rint(T x)
 #    endif
 #elif ARCH(AARCH64)
     AARCH64_INSTRUCTION(frintx, x);
+#elif ARCH(RISCV64)
```

Comment:
It's unfortunate that we have this both here and in LibC, but that predates this PR.

(It's also unfortunate that we have so much much inline asm in here. For rint, at least clang seems to DTRT for __builtin_rint for both x86_64 (with a new enough -march) and arm64, but not for riscv. Maybe I just need to tell the compiler that it can call this instruction here somehow though.

<details>
<summary>__builtin_rint test</summary>

```
% cat rint.c 
double rint(double d) {
    return __builtin_rint(d);
}

 % clang --target=aarch64-unknown-unknown -S -o - rint.c -O2 | rg -A4 rint:
rint:                                   // @rint
	.cfi_startproc
// %bb.0:
	frintx	d0, d0
	ret

% clang --target=x86_64-unknown-unknown -S -o - rint.c -O2 -march=core-avx2 | rg -A9 rint:
rint:                                   # @rint
	.cfi_startproc
# %bb.0:
	pushq	%rbp
	.cfi_def_cfa_offset 16
	.cfi_offset %rbp, -16
	movq	%rsp, %rbp
	.cfi_def_cfa_register %rbp
	vroundsd	$4, %xmm0, %xmm0, %xmm0
	popq	%rbp

% clang --target=x86_64-unknown-unknown -S -o - rint.c -O2 | rg -A11 rint:
rint:                                   # @rint
	.cfi_startproc
# %bb.0:
	pushq	%rbp
	.cfi_def_cfa_offset 16
	.cfi_offset %rbp, -16
	movq	%rsp, %rbp
	.cfi_def_cfa_register %rbp
	popq	%rbp
	.cfi_def_cfa %rsp, 8
	jmp	rint@PLT                        # TAILCALL
.Lfunc_end0:

 % out/gn/bin/clang --target=riscv64-unknown-unknown -S -o - rint.c -O2 | rg -A 5 rint:
rint:                                   # @rint
# %bb.0:                                # %entry
	addi	sp, sp, -16
	sd	ra, 8(sp)                       # 8-byte Folded Spill
	call	rint
	ld	ra, 8(sp)                       # 8-byte Folded Reload
```
</details>

Reply:
I made the LibC functions AK-backed now, which should at least be somewhat better.
But yeah having so much inline asm in AK isn't really nice.